src/rust/dbv2/migrations/2023-12-01-173351_add_total_fees/up.sql

```diff
@@ -0,0 +1,36 @@
+-- Your SQL goes here
+DROP VIEW api.orders;
+
+
+ALTER TABLE aggregator.user_history
+ADD COLUMN total_fees_paid_in_quote_subunits NUMERIC(20,0);
+
+
+WITH total_fees AS (
+    SELECT
+        SUM(taker_quote_fees_paid) AS total_fees_paid_in_quote_subunits,
+        taker_order_id,
+        market_id
+    FROM fill_events AS f
+    WHERE f.taker_address = f.emit_address
```

Comment:
How about switching to `f.maker_address`, since a fill event isn't always emitted to taker side (e.g. non-signing swap)?